### DISCUSSION TITLE
Terrakube and Terraform BSL license

### DISCUSSION THREAD

**Discussion by alfespa17** [terrakube-io/terrakube#467] [General]
Hello recently Hashicorp did a change in the [terraform license](https://github.com/hashicorp/terraform/blob/main/LICENSE), I sent an e-mail to check if Terrakube is compatible with it.

I got the following response from Hashicorp.

![image](https://github.com/AzBuilder/terrakube/assets/4461895/e2a7c578-a25d-4561-8083-d9c96f164073)

Terrakube is compatible with the new license 👍 so we will keep adding features.

Regards.

**Comment by jderieg** [comment]
You guys should also sign/pledge on the OpenTF manifesto at https://opentf.org/

**Reply by jcanizalez** [reply]
Thanks @jderieg  Terrakube has been signed the manifesto

**Comment by adevops01** [comment]
I have installed the terrakube on the kubernetes cluster but when i am trying to create an workspace i am getting this error. 
You are not authorized to create Workspace.
Please contact your administrator and request to include you in the Terrakube Administrator group.
Can you please help me fix this issue?

**Reply by alfespa17** [reply]
You need to go to the organization settings and add your team, even if you are inside the admin group you only have access to create teams and manage teams access, so you need to add the team to the org and grant permission to create workspaces
